requirements.txt

```diff
@@ -1 +1,5 @@
-Django==4.0.4
\ No newline at end of file
+Django==4.0.4
+djangorestframework
+djangorestframework-simplejwt
```

Comment:
How you have frozen the requirements? It is not containing the version of the package?